js/index.js

```diff
@@ -1,6 +1,51 @@
 // Iteration 1: Names and Input
-//
+
+
+let hacker1="Max";
+console.log(`The driver's name is ${hacker1}`);
+
+let hacker2="Anna";
+console.log(`The navigator's name is ${hacker2}`);
+
+
+
 // Iteration 2: Conditionals
 
+let stringshacker1=hacker1.length;
+let stringshacker2=hacker2.length;
+
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${stringshacker1} characters`);
+}
+
+else if (hacker1.length < hacker2.length) { 
+
+console.log(`It seems that the navigator has the longest name, it has ${stringshacker2} characters.`)
+}
+
+else if (hacker1.length === hacker2.length) {
+
+    console.log(`Wow, you both have equally long names, ${stringshacker1} characters!`)
+}
+
+
 
 // Iteration 3: Loops
+
+for (let i= 0 ; i < hacker1.length ; i++) {
```

Comment:
Here , we want to see a word with capital letters and with space between : "M A X" , not seprate letters , you need to declare a varible before loop, and add every letter in that :
let capitalName=""
for (let i= 0 ; i < hacker1.length ; i++) {
capitalName= capitalName+ hacker1[i]+" " 
  }
    console.log(capitalName.toUpperCase());